### PR TITLE
GCB-198: Add empty state ViewModel + header and subheader.

### PR DESCRIPTION
**Pull request by ochococo**
🚚  https://macoscope.atlassian.net/browse/GCB-198

Bonus:
- Changed ImageWidget background from flashing white to proper black.
- Improved handling of second render.
- After consulting with Dawid - added Header and Subheader label instead of dashed line in empty mode.

simulator screen shot 14 04 2016 19 58 47

### CODE REVIEW

File: GrandCentralBoard/Widgets/Harvest/HarvestWidget.swift [hunk 1]
```diff
@@ -25,20 +25,27 @@ final class HarvestWidget : Widget {
...
+        let items = [AreaBarItemViewModel(proportionalWidth: 0.5, proportionalHeight: 0.2, color: UIColor.lipstick(), valueLabelMode: .VisibleLabelLeft(text: "123")), AreaBarItemViewModel(proportionalWidth: 0.25, proportionalHeight: 0.5, color: UIColor.aquaMarine(), valueLabelMode: .VisibleWithHiddenLabel), AreaBarItemViewModel(proportionalWidth: 0.25, proportionalHeight: 1, color: UIColor.almostWhite(), valueLabelMode: .VisibleLabelRight(text: "222"))]
 
-        let model = AreaBarChartViewModel(barItems: items, horizontalAxisStops: 20, horizontalAxisLabelText: "people billing:", hotizontalAxisCountLabelText: "666")
+        let model = AreaBarChartViewModel(barItems: items, horizontalAxisStops: 20, horizontalAxisLabelText: "people billing:", hotizontalAxisCountLabelText: "666", centerText: nil, headerText: "HARVEST BURN REPORT", subheaderText: "Monday 28.03.2016")
 
         widgetView.render(model)
+
+        // Why another render? To check if view renders the view model after the previous one.
+        // TODO: Remove!
+        widgetView.render(AreaBarChartViewModel.emptyViewModel(header: "HARVEST BURN REPORT", subheader: "Monday 28.03.2016"))
```
Comment (michallaskowski) [inline GrandCentralBoard/Widgets/Harvest/HarvestWidget.swift]: When it should be removed? I am not sure what needs to happen to do that.

Comment (ochococo) [inline GrandCentralBoard/Widgets/Harvest/HarvestWidget.swift]: Karol will know.

Comment (michallaskowski) [inline GrandCentralBoard/Widgets/Harvest/HarvestWidget.swift]: Haha, good enough :)

**Comment by michallaskowski** [comment]
Ok, I have no more comments, merging. Good job

**Pull request merged**